config/file_config.go

```diff
@@ -350,6 +353,14 @@ func (f *fileConfig) validateGeneralConfigs() error {
 	default:
 		return fmt.Errorf("invalid CacheOverrunStrategy: '%s'", st)
 	}
+
+	stressReliefConfig := f.GetStressReliefConfig()
+	switch stressReliefConfig.Strategy {
```

Comment:
Instead of adding `Strategy` to the stress relief config struct can we use `f.GetCacheOverrunStrategy()` and validate that the result is `impact` when `f.GetStressReliefConfig().mode` is `monitor` or `always`?

Reply:
yes, I overcomplicated looking at the cache overrun strategy. Fix incoming.